GitCommands/Git/Executable.cs

```diff
@@ -230,12 +231,15 @@ public StreamReader StandardError
             public void WaitForInputIdle() => _process.WaitForInputIdle();
 
             /// <inheritdoc />
-            public Task<int> WaitForExitAsync() => _exitTaskCompletionSource.Task;
+            public Task<int> WaitForExitTaskAsync() => _exitTaskCompletionSource.Task;
```

Comment:
Similarly this should be `WaitForExitAsync`. Async methods don't generally contain the word 'task'.

Reply:
Async or not is the same.
The name is for the exitTask.